host/class/msc/usb_host_msc/include/usb/msc_host_vfs.h

```diff
@@ -16,6 +16,16 @@ extern "C" {
 
 typedef struct msc_host_vfs *msc_host_vfs_handle_t;           /**< VFS handle to attached Mass Storage device */
 
+/**
+ * @brief Format MSC device.
+ *
+ * @param[in] device Device handle obtained from MSC callback provided upon initialization
+ * @param[in] mount_config Mount configuration
+ * @param[in] vfs_handle Handle to MSC device associated with registered VFS
+ * @return esp_err_t
+ */
+esp_err_t esp_vfs_fat_usb_msc_format(msc_host_device_handle_t device, const esp_vfs_fat_mount_config_t *mount_config, const msc_host_vfs_handle_t vfs_handle);
```

Comment:
Usually, all the function names have the same pattern inside the one file. 
Here, we have two public calls: 
- `msc_host_vfs_register`
- `msc_host_vfs_unregister`

So if we want to provide a new API within current module/files, I would assume that it should be something like: 
```suggestion
esp_err_t msc_host_vfs_format(msc_host_device_handle_t device, const esp_vfs_fat_mount_config_t *mount_config, const msc_host_vfs_handle_t vfs_handle);
```

Reply:
@roma-jam The naming was actually my idea, I wanted to make it similar to other formatting functions that we have in esp-idf, eg sdcard_format

https://github.com/espressif/esp-idf/blob/a2f420a36ecf8cd19b443adde1d75e5eaecc6309/components/fatfs/vfs/esp_vfs_fat.h#L268

I'd like to rename `msc_host_vfs_register/unregister` accordingly too... WDYT?

Reply:
@tore-espressif Oh, I see. 

For `esp_vfs_fat` it makes sense, as the component itself has the similar name. 

If apply the same logic on our case .. it should be `esp_usb_msc_host_format` or something like that.

Reply:
OK, let's go for consistency within the module, so `msc_host_vfs_format`